sdk/storage/service.projects

```diff
@@ -0,0 +1,26 @@
+<!--
+  This file allows overrides and extensions to the build configuration defined
+  by the central engineering system, and is considered during pipeline builds for
+  CI, test runs, and package release.
+
+  Based on the SDKType variable (msbuild property or env variable) this will filter
+  the set of projects to to match what type of SDK we are interested in building.
+
+  SDKType values
+   - "all" will build all projects
+   - "dataplane" will build the new track 2 client/data libraries
+   - "mgmt" will build the management plane SDK.
+   - "webjobs" will build the storage WebJobs Extensions.
+-->
+
+<Project>
+  <ItemGroup Condition="'$(SDKType)' == 'dataplane'">
+    <ProjectsToIncludeBySDKType Include="$(MSBuildThisFileDirectory)Azure.Storage.*\**\*.csproj" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(SDKType)' == 'mgmt'">
+    <ProjectsToIncludeBySDKType Include="$(MSBuildThisFileDirectory)Azure.ResourceManager.*\**\*.csproj" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(SDKType)' == 'webjobs'">
```

Comment:
@jsquire should I change this to 'functions' to match the ci.functions.yaml file?

Reply:
Yes.   We'll also need one for `datamovement`

Reply:
Done.